app/views/publications/mid_cycle_report/show.html.erb

```diff
@@ -0,0 +1,211 @@
+            <li>
+              <a href="#offer" class="govuk-link app-toc__link">
+                <span class="app-toc__number">4.</span>
+                <span class="app-toc__content">Applications with an offer</span>
+              </a>
+            </li>
+            <li>
+              <a href="#accepted" class="govuk-link app-toc__link">
+                <span class="app-toc__number">5.</span>
+                <span class="app-toc__content">Offers accepted</span>
+              </a>
+            </li>
+            <li>
+              <a href="#deferrals" class="govuk-link app-toc__link">
+                <span class="app-toc__number">6.</span>
+                <span class="app-toc__content">Deferrals</span>
+              </a>
+            </li>
+            <li>
+              <a href="#rejected" class="govuk-link app-toc__link">
+                <span class="app-toc__number">7.</span>
+                <span class="app-toc__content">Applications rejected</span>
+              </a>
+            </li>
+            <li>
+              <a href="#automatically" class="govuk-link app-toc__link">
+                <span class="app-toc__number">8.</span>
+                <span class="app-toc__content">Applications rejected automatically</span>
+              </a>
+            </li>
+            <li>
+              <a href="#proportion" class="govuk-link app-toc__link">
+                <span class="app-toc__number">9.</span>
+                <span class="app-toc__content">Proportion of applications rejected</span>
+              </a>
+            </li>
+          </ol>
+        </div>
+      </div>
+    </div>
+  </div>
+
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+      <h2 class="govuk-heading-m" id="data">How this data compares to other data we publish</h2>
+      <p class="govuk-body">The page showing <%= govuk_link_to('monthly statistics for courses starting in the 2023 to 2024 academic year', publications_monthly_report_itt_path(RecruitmentCycle.current_year)) %>, calculates data using the same method UCAS used to. This method counts a candidate’s furthest application. For example, if an application moves from ‘recruited’ to ‘deferred’ the recruited count will go down, and the deferred count will go up. This method does not easily show where a candidate ends up over time.</p>
+      <p class="govuk-body">This page about national recruitment performance, calculates data in a new way by counting all submitted applications, regardless of their outcome. It also counts all offers regardless of if the offer is accepted, declined or waiting for a decision.</p>
+      <p class="govuk-body">With each page using a different method, you should not compare this page with the monthly statistics page.</p>
+    </div>
+  </div>
+
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+      <h2 class="govuk-heading-m" id="submitted">2. Candidates who’ve submitted applications</h2>
```

Comment:
I'm hoping this is the last content change! But for this table, where we only show Primary or Secondary. Can we change the table header from 'Subject' back to '**Level**'? As I don't think subject makes sense for this one.